src/mumble/ConnectDialog.cpp

```diff
@@ -1823,3 +1832,14 @@ void ConnectDialog::fetched(QByteArray xmlData, QUrl, QMap<QString, QString> hea
 
 	fillList();
 }
+
+void ConnectDialog::on_qleServerFilter_textChanged(const QString &filter)
+{
+	ServerItem::filter = filter;
+	foreach(ServerItem *si, qlItems) {
+		if (si->nameMatchesFilter())
```

Comment:
I think instead of having a static `filter` variable it'd be better to pass the current filter text as a parameter to `nameMatchesFilter` like so:
```suggestion
	foreach(ServerItem *si, qlItems) {
		if (si->nameMatchesFilter(filter))
```